src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! Approximate Lower Bound Arguments (ALBA, <https://eprint.iacr.org/2023/1655.pdf>)
+//!
+//! Alba is a generic protocol to prove succinctly a lower bound of the size of
+//! a, potentially weighted, set. Say we have a set Sp of size |Sp| >= $n_p$,
```

Comment:
by a `potentially weighted` set do you simply mean `multiset`?

Reply:
Do we actually need a description of alba here? We have README, the website and @curiecrypt is working on more documentation.

Reply:
Let's agree with @tolikzinovyev and keep all documentation in the README and website.